pkgs/applications/networking/mullvad-vpn/default.nix

```
@@ -1,145 +1,81 @@
-{ stdenv
-, lib
-, fetchurl
-, dpkg
-, alsa-lib
-, atk
-, cairo
-, cups
-, dbus
-, expat
-, fontconfig
-, freetype
-, gdk-pixbuf
-, glib
-, pango
-, nspr
-, nss
-, gtk3
-, mesa
-, libGL
-, wayland
-, xorg
-, autoPatchelfHook
-, systemd
-, libnotify
-, libappindicator
-, makeWrapper
-, coreutils
-, gnugrep
+{
+  lib,
+  buildNpmPackage,
+  protobuf,
+  protoc-gen-js,
+  grpc-tools,
+  makeWrapper,
+  electron,
+  mullvad,
 }:
 
-let
-  deps = [
-    alsa-lib
-    atk
-    cairo
-    cups
-    dbus
-    expat
-    fontconfig
-    freetype
-    gdk-pixbuf
-    glib
-    pango
-    gtk3
-    libappindicator
-    libnotify
-    mesa
-    xorg.libX11
-    xorg.libXScrnSaver
-    xorg.libXcomposite
-    xorg.libXcursor
-    xorg.libXdamage
-    xorg.libXext
-    xorg.libXfixes
-    xorg.libXi
-    xorg.libXrandr
-    xorg.libXrender
-    xorg.libXtst
-    xorg.libxcb
-    xorg.libxshmfence
-    nspr
-    nss
-    systemd
-  ];
+buildNpmPackage rec {
+  pname = "mullvad-vpn";
+  inherit (mullvad) src version;
```

```suggestion
  inherit (mullvad) src version meta;
```
Very optional: Inherit the `meta` attributes from the main package and use `//` set updates when needed.